swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = None, condition = None, age = None):
+        super().__init__(category, condition, age)
+        self.category = "Clothing"
```

Comment:
I like that you have hardcoded "clothing" here since the child class Clothing will always have the category "Clothing," but we should give that to the `super`. category isn't a new attribute specific to the child class.

```suggestion
    def __init__(self, condition = None, age = None):
        super().__init__("Clothing", condition, age)
```

now the user can't have control over the category and mess it up haha